rfc.tex

```diff
@@ -11,7 +11,7 @@ \section{Decision Making Process}
 
 [show Jira RFC workflow here?]
 
-In 20?? we decided to change our approach to the latter process.
+In 2014 we decided to change our approach to the latter process.
 DM developers have to be familiar with Jira and we developed a Jira workflow for ``Request for Comments'' (RFCs).
 RFCs were designed to empower individual developers to propose decisions.
 Emphasis is placed on the developer being responsible for any fall out from a change.
```

Comment:
This is a bit misleading, and the next sentence is false.  RFCs can be used to propose work for others; the main point was supposed to be that if you're going to do all the work yourself, you don't even need an RFC (unless it affects others).

Reply:
I really thought that we preferred RFCs where the proposer was going to take responsibility for implementation.